crawler.py

```diff
@@ -0,0 +1,91 @@
+""" Async Web Crawler """
```

Comment:
The module that you run by convention should be `main.py`. Rename the module name acoordingly.